python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -60,11 +60,6 @@ def test_repartitionByRange_dataframe(self):
     def test_repr_behaviors(self):
         super().test_repr_behaviors()
 
-    # TODO(SPARK-41874): Implement DataFrame `sameSemantics`
```

Comment:
```
Traceback (most recent call last):
  File "/__w/spark/spark/python/pyspark/sql/tests/test_dataframe.py", line 1494, in test_same_semantics_error
    with QuietTest(self.sc):
AttributeError: 'DataFrameParityTests' object has no attribute 'sc'
```

I think you can not enable this test since we don't have `SparkContext`, but you can remove the TODO and update it to 
```
    @unittest.skip("Spark Connect does not SparkContext but the tests depend on them.")
    def test_same_semantics_error(self):
        super().test_same_semantics_error()
```

Reply:
I see. Thanks!